mne/connectivity/cfc.py

```diff
@@ -0,0 +1,383 @@
+    ixs : array-like, shape (n_pairs x 2)
+        The indices for low/high frequency channels. PAC will be estimated
+        between n_pairs of channels. Indices correspond to rows of `data`.
+    pac_func : string, ['plv', 'glm', 'mi_canolty', 'mi_tort', 'ozkurt']
+        The function for estimating PAC. Corresponds to functions in pacpy.pac
+    ev : array-like, shape (n_events,) | None
+        Indices for events. To be supplied if data is 2D and output should be
+        split by events. In this case, tmin and tmax must be provided
+    tmin : float | None
+        If ev is not provided, it is the start time to use in inst. If ev
+        is provided, it is the time (in seconds) to include before each
+        event index.
+    tmax : float | None
+        If ev is not provided, it is the stop time to use in inst. If ev
+        is provided, it is the time (in seconds) to include after each
+        event index.
+    n_jobs : int
+        Number of CPUs to use in the computation.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    pac_out : array, dtype float, shape (n_pairs, [n_events])
+        The computed phase-amplitude coupling between each pair of data sources
+        given in ixs.
+    """
+    from ..externals.pacpy import pac as ppac
+    func = getattr(ppac, pac_func)
+    ixs = np.array(ixs, ndmin=2)
+
+    if data.ndim < 2 or data.ndim > 3:
+        raise ValueError('Data must be 2D or 3D')
+    if data.ndim == 2:
+        data = data[np.newaxis, ...]
+    if ixs.shape[1] != 2:
+        raise ValueError('Indices must have have a 2nd dimension of length 2')
+    if len(f_phase) != 2 or len(f_amp) != 2:
+        raise ValueError('Frequencies must be specified w/ a low/hi tuple')
+    if pac_func not in ppac.__dict__.keys():
+        raise ValueError("That PAC function doesn't exist in PacPy")
+
+    n_epochs, n_channels, n_times = data.shape
+    parallel, my_pac, n_jobs = parallel_func(_my_pac, n_jobs=n_jobs,
+                                             verbose=verbose)
+    pacs = np.array(parallel(my_pac(data, ixf, ixa, f_phase, f_amp, func,
+                                    ev=ev, tmin=tmin, tmax=tmax, sfreq=sfreq)
+                             for ixf, ixa in ixs))
+    pacs = np.atleast_2d(pacs)
+    return pacs
+
+
+def _filter_ph_am(xph, xam, f_ph, f_am, sfreq, filterfn=None, kws_filt=None):
+    """Aux function for phase/amplitude filtering"""
+    from ..externals.pacpy.pac import _range_sanity
+    from scipy.signal import hilbert
+    filterfn = band_pass_filter if filterfn is None else filterfn
+    kws_filt = {} if kws_filt is None else kws_filt
+
+    # Filter the two signals + hilbert/phase
+    _range_sanity(f_ph, f_am)
+    xph = filterfn(xph, sfreq, *f_ph)
+    xam = filterfn(xam, sfreq, *f_am)
+
+    xph = np.angle(hilbert(xph))
+    xam = np.abs(hilbert(xam))
```

Comment:
Passing the second argument to hilbert makes it faster (at least in some cases). I mean -  the next power of 2 with respect to data length. Again - this is performance-related so it can wait.